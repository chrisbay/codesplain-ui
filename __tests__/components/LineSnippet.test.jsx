import React from 'react'
import ReactTestUtils from 'react-addons-test-utils';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

import LineSnippet from '../../src/components/LineSnippet';

describe('<LanguageSelector />', () => {
  it('matches snapshot', () => {
    const renderer = ReactTestUtils.createRenderer();
    const tree = renderer.render(
      <MuiThemeProvider>
        <LineSnippet
          lineNumber={1}
          value={'foo bar baz'}
        />
      </MuiThemeProvider>
    );
    expect(tree).toMatchSnapshot();
  })
})
