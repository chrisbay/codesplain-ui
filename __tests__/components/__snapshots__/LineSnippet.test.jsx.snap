exports[`<LanguageSelector /> matches snapshot 1`] = `
<LineSnippet
  lineNumber={1}
  value="foo bar baz" />
`;
